cookiecutter/local_extensions.py

```diff
@@ -123,6 +133,19 @@ def _get_random_weekday(self) -> str:
             ]
         )
 
+    def _get_copyright_year(self) -> int:
+        """Get the copyright year.
+
+        This will be faked when running the golden tests to make sure the tests are
+        reproduceable.
+
+        Returns:
+            The copyright year.
+        """
+        if self._is_golden_testing():
+            return 2023
+        return _datetime.datetime.now().year
```

Comment:
Optionally we could have added `timezone` but it's not a big deal

Reply:
No, we need the local time here, but it could have a comment to clarify.